### ISSUE TITLE
Read charging status

### ISSUE THREAD

**Issue by sveip** [wido/smarthpwc#1]
Hi @wido, thanks for the project. 

To accomplish load balancing, feedback from the slaves on charging presence and possibly rate is needed.

Have you investigated if that is possible? 

Idea if possible: Read max amp car is able to receive, sum these numbers and divide on number of slaves, and set that as max charge current to prevent overloading fuse.

**Comment by wido** [comment]
Thanks! I still have to work on this and have given this a low priority on my side. I'll probably pick up on this later this year

**Comment by sveip** [comment]
Great :) Do you know if there is a protocol description and/or API somewhere, I might get time to have a look.

**Comment by wido** [comment]
Nope, not really. You will mainly need to dig into the RS485 messages

**Comment by sveip** [comment]
I see. I only have one TWC atm., so I guess that will not be super easy ;)

2018-06-04 9:18 GMT+02:00 Wido den Hollander <notifications@github.com>:
